global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview6-011773",
+    "dotnet": "3.0.100-preview6-012105",
```

Comment:
This version of .NET SDK has a MSBuild fix for `GetPathsOfAllDirectoriesAbove`

Reply:
Please update the contributing docs to link to the new SDK version and installer link. 

---
In reply to: [288795365](https://github.com/dotnet/roslyn/pull/36042#discussion_r288795365) [](ancestors = 288795365)